apps/drec-api/src/guards/NotDeletedUserGuard.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Injectable,
+  CanActivate,
+  ExecutionContext,
+  HttpException,
+} from '@nestjs/common';
+import { UserStatus } from '@energyweb/origin-backend-core';
+
+@Injectable()
+export class NotDeletedUserGuard implements CanActivate {
```

Comment:
To be honest I think deleted user should not be able to perform any action and it should be built in normal auth guard

Reply:
Yeah, and we already had an Active user guard. So I just updated that and removed this one.